app/controllers/sites_controller.rb

```diff
@@ -0,0 +1,31 @@
+class SitesController < ApplicationController
+  before_action :set_site
+  load_and_authorize_resource
```

Comment:
I figured there was value in consistency; cancancan will short-circuit when it's already loaded, so it isn't harmful.